apps/meteor/app/api/server/v1/rooms.ts

```diff
@@ -210,62 +211,68 @@ API.v1.addRoute(
 				return API.v1.unauthorized();
 			}
 
-			const file = await getUploadFormData(
+			const file1 = await getUploadFormData(
```

Comment:
This API should not accept multiple files. Please call this API multiple times, one for each file, to upload multiple files.

Reply:
Yes sir i am working on it

Reply:
sir i have added a commit of sharing one file at a time